### PR TITLE
updated various stlying and layout per notion feedback

### PR DESCRIPTION
**Pull request by cmusson**

### CODE REVIEW

File: web/app.fluidity.money/app/components/ViewRewardModal.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const ViewRewardModal = ({
 }: IPropsConnectedWalletModal) => {
   const [modal, setModal] = useState<React.ReactPortal | null>(null);
   const { width } = useViewport();
-  const isMobile = width < 500;
+  const isMobile = width < 500 && width > 0;
```
Comment (glo12345) [inline web/app.fluidity.money/app/components/ViewRewardModal.tsx:40]: Why do we need to check for width >0?

File: web/app.fluidity.money/app/routes/$network/dashboard/home.tsx [hunk 2]
```diff
@@ -313,8 +313,8 @@ export default function Home() {
   );
 
   const { width } = useViewport();
-  const isTablet = width < 850;
-  const isMobile = width < 500;
+  const isTablet = width < 850 && width > 500;
```
Comment (glo12345) [inline web/app.fluidity.money/app/routes/$network/dashboard/home.tsx]: Keep the original isMobile and isTablet - if isTable is true, it also implies isMobile is true

**Review by glo12345 — DISMISSED** [review]

File: web/app.fluidity.money/app/components/ViewRewardModal.tsx (hunk 1 above)
Comment (cmusson) [reply web/app.fluidity.money/app/components/ViewRewardModal.tsx:40]: It's related to server side rendering and the hook reading the screen size as 0 initially then updating to the actual screen size. It's been causing components to render on first load for a split second then disappear. This stops that.

File: web/app.fluidity.money/app/routes/$network/dashboard/home.tsx (hunk 2 above)
Comment (cmusson) [reply web/app.fluidity.money/app/routes/$network/dashboard/home.tsx]: ok!

**Review by joshocalico — APPROVED** [review]

**Pull request merged**